crt_portal/cts_forms/tests.py

```diff
@@ -109,3 +87,52 @@ def test_required_where(self):
             'public_or_private_healthcare': 'state_local_facility',
         })
         self.assertTrue('place<ul class="errorlist"><li>This field is required.' in str(form.errors))
+
+
+class ContactValidationTests(TestCase):
+    def test_non_ascii_name(self):
+        form = Contact(data={
+            'contact_first_name': '李王',
+            'contact_last_name': '王-Núñez',
+            'contact_email': '',
+            'contact_phone': ''
+        })
+        self.assertTrue(form.is_valid())
+
+    def test_non_ascii_email(self):
+        form = Contact(data={
+            'contact_first_name': '',
+            'contact_last_name': '',
+            'contact_email': 'foo@bär.com',
+            'contact_phone': ''
+        })
+        self.assertTrue(form.is_valid())
+
+    def test_phone_too_short(self):
+        """Model validation tests need to be structured differently"""
```

Comment:
@LindsayYoung Can you say more about this code comment? Not sure what it's saying.

Reply:
Would this be better: """Model validation unit tests require testing the model directly"""
I am trying to and an explanation as to why this looks different from the other tests.

Reply:
Ah, that's helpful! I thought the earlier comment was saying something like, "this needs to be changed sometime down the line."